tests/MemoryHandlerTest.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by chad_olsen on 2017/06/02.
+ */
+public class MemoryHandlerTest {
```

Comment:
Are these tests and therefore the MemoryHandler class not redundant because when we read and write accounts we are reading and writing their specific Catalogue with the account and not in a separate file?